flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/data/vector/VectorizedColumnBatchTest.java

```diff
@@ -176,8 +176,7 @@ public TimestampData getTimestamp(int colId, int precision) {
                             julianDay |= (bytes[i] & (0xff));
                         }
                         long millisecond =
-                                (julianDay - DateTimeUtils.EPOCH_JULIAN)
-                                                * DateTimeUtils.MILLIS_PER_DAY
+                                (julianDay - 2440588) * DateTimeUtils.MILLIS_PER_DAY
```

Comment:
Why replace the constant with a literal? If you make the constant `private` I'd suggest to extract this computation in a utility method, even if it's only for testing.